### PR TITLE
Itp 1362 web mobile form to add a new card

### PR DESCRIPTION
**Pull request by Abdulla-ab**
[ITP-1362]
8ADA2906-4F77-4428-B444-6EC22615D97B

[ITP-1362]: https://fcodelabs.atlassian.net/browse/ITP-1362?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export class HomeComponent implements OnInit {
+  public form: FormGroup = new FormGroup({
+    title: new FormControl(),
+    description: new FormControl(),
+  });
+
+  public submit(): void {
+    this.form.markAllAsTouched();
+    if (this.form.valid) {
+      console.log(this.form.value);
```
Comment (ChamaraWee) [inline src/app/home/home.component.ts:18]: remove logs

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent implements OnInit {
+  public form: FormGroup = new FormGroup({
+    title: new FormControl(),
```
Comment (ChamaraWee) [inline src/app/home/home.component.ts:11]: use validators here

File: src/app/home/home.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    description: new FormControl(),
+  });
+
+  public submit(): void {
+    this.form.markAllAsTouched();
```
Comment (ChamaraWee) [inline src/app/home/home.component.ts:16]: any specific reason for this?

**Review by ChamaraWee — CHANGES_REQUESTED** [review]

File: src/app/home/home.component.ts (hunk 3 above)
Comment (Abdulla-ab) [reply src/app/home/home.component.ts:16]: to ensure that all fields have been touched and properly filled out before submitting the form.

**Pull request closed without merging**